src/scripts/gym/GymList.ts

```diff
@@ -1589,21 +1589,41 @@ GymList['Friend League'] = new Gym(
 GymList['Quick League'] = new Gym(
     'Koichiro',
     'Quick League',
-    [new GymPokemon('Magikarp', 1, 20)],
+    [new GymPokemon('Magikarp Skelly', 1, 20)],
     BadgeEnums.Quick_League,
     0, //TODO
     'TODO',
-    [new RouteKillRequirement(10, GameConstants.Region.alola, 32)]
+    [new RouteKillRequirement(10, GameConstants.Region.alola, 32)],
+    () => {
+        if (!App.game.party.alreadyCaughtPokemonByName('Magikarp Skelly')) {
+            Notifier.notify({
+                message: 'You were awarded a Magikarp Skelly!',
+                type: NotificationConstants.NotificationOption.success,
+                setting: NotificationConstants.NotificationSetting.General.new_catch,
+            });
+            App.game.party.gainPokemonById(pokemonMap['Magikarp Skelly'].id);
```

Comment:
Probably want to pass `suppressNotification = true` for these, getting two notifications which basically say the same thing right now.
Also this doesn't look like it will roll for shiny, do we want it to? I would probably have gainPokemonById handle the generation like it does for gender, if we explicitly pass `shiny = null` or something. Or just leave it always be not shiny.

Reply:
Fixed